packages/st2/debian/postinst

```diff
@@ -53,6 +53,7 @@ case "$1" in
       # Initially set destination files owenership (only on the first install)
       [ "$upgrading" = 1 ] || set_permissions "$SET_PERMS"
       rm -f $ST2_UPGRADESTAMP
+      systemctl daemon-reload
```

Comment:
would be nice to include a small code comment why we're doing this. Otherwise, it might hard to get a reason after some time.